src/pages/index.tsx

```diff
@@ -58,6 +64,12 @@ export default function IndexPage() {
     { callWithTruthyDeps: true }
   )
   const [ subscriptions, subscriptionsState ] = useSubscriptions()
+  const responsive = useResponsive()
+  const isMobile = responsive({ maxWidth: Responsive.onlyMobile.maxWidth })
+  const burgerMenu = useContext(BurgerMenuStatusContext)
+  const burgerShow = useContext(BurgerMenuShowContext)
+  const translate = '550px'
```

Comment:
`translate` can be outside the component. Also what does this number means? Is it some element height? Can we reuse this variable?

Reply:
this got refactored in an [upcoming PR](https://github.com/decentraland/governance/pull/186)